Makefile

```diff
@@ -77,7 +77,6 @@ clean:
 lint:
 	flake8 *.py $(APP_NAME) tests
 	mypy $(APP_NAME) --ignore-missing-imports
-	unset TF_CLI_ARGS_init; cd tests/terraform; terraform init; terraform validate
```

Comment:
This looks like it should be added back in. Accidental remove?

Reply:
I found that the only way to run the tests successfully was to remove this line